src/scss/utils/theme-styles.scss

```diff
@@ -43,7 +43,8 @@ $default-theme-colors: (
 
 $default-light-colors: (
   "light-grey",
-  "white"
+  "white",
+  "default-background",
```

Comment:
I think this broke the footer in the default background? Left is prod, right is pull request.
![image](https://user-images.githubusercontent.com/111025/62094931-d9e43a00-b2d2-11e9-9ca5-feeac651bf11.png)

Reply:
Done